webknossos/__generate_client.py

```diff
@@ -17,7 +19,7 @@
 from webknossos.client.context import _get_generated_client
 from webknossos.utils import snake_to_camel_case
 
-SCHEMA_URL = "https://converter.swagger.io/api/convert?url=https%3A%2F%2Fwebknossos.org%2Fswagger.json"
+SCHEMA_URL = "https://converter.swagger.io/api/convert?url=https%3A%2F%2Fmaster.webknossos.xyz%2Fswagger.json"
```

Comment:
The webknossos.org schema does not contain the `/data` routes anymore, since it does not have an internal datastore anymore. I used the dev-deployment as a workaround for now.